lib/matplotlib/patches.py

```diff
@@ -750,9 +763,35 @@ def get_patch_transform(self):
         # important to call the accessor method and not directly access the
         # transformation member variable.
         bbox = self.get_bbox()
-        return (transforms.BboxTransformTo(bbox)
-                + transforms.Affine2D().rotate_deg_around(
-                    bbox.x0, bbox.y0, self.angle))
+        if self.rotation_point == 'center':
+            width, height = bbox.x1 - bbox.x0, bbox.y1 - bbox.y0
```

Comment:
Do bounding boxes not have width/height properties?

Reply:
No, but there is `self.get_width()` and `self.get_height()`, which seem to be invariant under rotation just as `bbox`.